Patch to fix building with GCC 4.1 when using X.
Thanks to Mark Loeser for providing the patch in bug #126593 and to
Matteo Azzali for reporting.

Index: xawtv-3.95/x11/vbi-gui.c
===================================================================
--- xawtv-3.95.orig/x11/vbi-gui.c
+++ xawtv-3.95/x11/vbi-gui.c
@@ -44,6 +44,7 @@
 #include "vbi-x11.h"
 #include "vbi-gui.h"
 
+#include "grab-ng.h"
 #include "channel.h"
 
 static int tt_debug = 1;
